Makefile

```
@@ -44,11 +44,9 @@ LINUX_BUILDDIR=$(BUILDDIR)/linux
 
 linux: $(LINUX_BUILDDIR)/tun2socks
 
-$(LINUX_BUILDDIR)/tun2socks: $(XGO)
-	mkdir -p "$(LINUX_BUILDDIR)/$(IMPORT_PATH)"
-	$(XGO) -ldflags $(XGO_LDFLAGS) --targets=linux/amd64 -dest "$(LINUX_BUILDDIR)" -pkg $(ELECTRON_PKG) .
-	mv "$(LINUX_BUILDDIR)/$(IMPORT_PATH)-linux-amd64" "$@"
-	rm -r "$(LINUX_BUILDDIR)/$(IMPORT_HOST)"
+$(LINUX_BUILDDIR)/tun2socks:
+	mkdir -p "$(@D)"
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=1 CC=x86_64-linux-musl-gcc go build --ldflags=--extldflags=$(XGO_LDFLAGS) -o "$@" $(ELECTRON_PKG)
```

Shall we statically link the musl-libc library using `...-musl-gcc -static ...` as mentioned in [How to Use musl](https://www.musl-libc.org/how.html)? I don't know whether `musl-gcc` without `-static` requires some musl dynamic library like `libmusl.so` to be installed in the final user's machine.

```suggestion
	GOOS=linux GOARCH=amd64 CGO_ENABLED=1 CC="x86_64-linux-musl-gcc -static" go build --ldflags=--extldflags=$(XGO_LDFLAGS) -o "$@" $(ELECTRON_PKG)
```